language/proto/generate_test.go

```diff
@@ -188,6 +188,84 @@ func TestGeneratePackage(t *testing.T) {
 	}
 }
 
+// Test generated files that have been consumed by other rules should not be
+// added to the  rule
```

Comment:
A doc comment should start with "TestConsumedGenFiles checks that ...". Remove the double space and put a period at the end of the sentence.